demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -686,7 +686,7 @@ static int establishMqttSession( MQTTContext_t * pMqttContext,
     if( mqttStatus != MQTTSuccess )
     {
         returnStatus = EXIT_FAILURE;
-        LogError( ( "MQTT init failed with status %u.", mqttStatus ) );
+        LogError( ( "MQTT init failed with status %s.", MQTT_Status_strerror( mqttStatus ) ) );
```

Comment:
(Nit) Log messages could be crisper (to follow the logging message format) as
```suggestion
        LogError( ( "MQTT init failed: Status=%s.", MQTT_Status_strerror( mqttStatus ) ) );
```